lib/ansible/modules/packaging/language/pip.py

```diff
@@ -235,6 +244,95 @@
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
 
+class Distribution:
+
+    """Wrapper class for pkg_resources.Requirement.
+
+    A wrapper class for Requirement, provide usefual API to parse package name and version specifier,
+    do fallback if the installed setuptools version doesn't support some features.
+
```

Comment:
Normally, it's a style violation to have an empty string at the end of docstring.

Reply:
I may have read a fake document...